packages/zero-cache/src/services/view-syncer/pipeline-driver.ts

```diff
@@ -571,7 +571,9 @@ class Streamer {
       switch (type) {
         case 'add':
         case 'remove': {
-          yield* this.#streamNodes(queryHash, schema, type, [change.node]);
```

Comment:
Are you sure this is even necessary? I haven't looked at impl of `streamNodes` but in similar code I remember realizing/proving to myself that the sql execute never even happens if nobody ever iterates the first time.

Reply:
That's why, for example, we just went ahead and setup the child relationships for each row "eagerly". Because we knew the sql statement to actually get them would never happen unless some downstream operator needed them.

Reply:
This might avoid the work of creating the query plan since we do not prepare the statement.

@grgbkr - was there a measurable difference with this change?